components/VueLayerMarkerPopup.vue

```diff
@@ -60,6 +61,39 @@ export default defineComponent({
         this.display = false;
         this.position = [0, 0];
       });
+      /**
+       * when any coordinates are updated, we check if the updated data is linked to the current popup.
+       * If it is, and we're displaying the popup, we also need to update its location.
+       */
+      eventBus.$on("newCoordinates", (data: Position) => {
+        if (this.details && this.display) {
+          // if the data ID is the same as the current popup ID
+          if (this.details.ID === data.id) {
+            // We update the position
+            this.position = [data.longitude, data.latitude];
+            // convert timestamp to readable format
+            const timestamp : Date = new Date(data.timestamp * 1000);
+            const tsString : string = timestamp.toLocaleString();
+            // update popup details when the attached asset has sent an update
+            const details = Object.fromEntries(Object.entries({
```

Comment:
This is the same code as in the VueLayerMap, maybe have them both handled at the same place, probably in VueLayerMarkerPopup makes the most sense.

Reply:
Good idea. Will look into it!